### PR TITLE
fixing broken link

### PR DESCRIPTION
**Pull request by hallaroo**
Fixing one link that 404's

### CODE REVIEW

File: content/en/docs/16.0/reference/programs/vtctldclient/vtctldclient_TabletExternallyReparented.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Updates the topology record for the tablet's shard to acknowledge that an extern
 
 Updates the topology record for the tablet's shard to acknowledge that an external tool made this tablet the primary.
 
-See the Reparenting guide for more information: https://vitess.io/docs/user-guides/reparenting/#external-reparenting.
+See the Reparenting guide for more information: https://vitess.io/docs/user-guides/configuration-advanced/reparenting/#external-reparenting.
```
Comment (ajm188) [inline content/en/docs/16.0/reference/programs/vtctldclient/vtctldclient_TabletExternallyReparented.md:13]: This is autogenerated from the code (https://github.com/vitessio/vitess/blob/main/go/cmd/vtctldclient/command/reparents.go#L291-L293)

Can you fix it there, then run (in this repo) `make vtctldclient-docs`?

**Review by ajm188 — DISMISSED** [review]
right change, just need to make it in a different spot (and fix it everywhere!)

**Comment by deepthi** [comment]
@ajm188 can you take care of this?

**Comment by ajm188** [comment]
https://github.com/vitessio/vitess/pull/13350

**Comment by hallaroo** [comment]
Thank you! And apologies for not responding sooner! This was on my get to this eventually list so thank you again for dealing with it.

**Comment by ajm188** [comment]
oh no i rebased when i meant to merge 🙈

**Review by deepthi — APPROVED** [review]

**Pull request merged**